presto-docs/src/main/sphinx/release/release-0.285.rst

```diff
@@ -2,6 +2,10 @@
 Release 0.285
 =============
 
+.. warning::
+
+   There is a bug of LIKE pattern with multiple-byte characters. (:pr:`21578`)
```

Comment:
`bug for`?

Reply:
@steveburnett can you comment on this?

Reply:
```suggestion
   There is a bug for LIKE pattern with multiple-byte characters. (:pr:`21578`)
```
Good catch @majetideepak! I didn't compare the title of the linked PR to the text on this page, I'd checked it on the newest (release-0.285.1.rst). 

It should be "for" on this page as well. Thanks!

Reply:
updated